src/main/java/seedu/address/ui/BookListPanel.java

```diff
@@ -11,7 +11,7 @@
 import seedu.address.model.book.Book;
 
 /**
- * Panel containing the list of persons.
+ * Panel containing the list of patrons.
```

Comment:
@yl-ang shldn't this be books?

Reply:
Yup. Have updated this in the open PR earlier today!